.node-version

```diff
@@ -1 +0,0 @@
-6.2.2
```

Comment:
Is there a reason you removed this `.node-version`? You should have something there.

Reply:
Oh wait. My b.